homeassistant/components/homematic/__init__.py

```diff
@@ -392,7 +393,7 @@ def setup(hass, config):
     for sname, sconfig in conf[CONF_HOSTS].items():
         remotes[sname] = {
             "ip": sconfig.get(CONF_HOST),
-            "port": DEFAULT_PORT,
+            "port": sconfig.get(CONF_PORT),
```

Comment:
Please use `dict[key]` for items with default config schema values.

Reply:
I followed the style of the rest of the file regarding the config items since it uses the get-method for all config items including the ones with defaults.

I guess it would look a bit out of place if I make this the only item in a different style, but if you like it this way I would change it to this?
```
"port": sconfig[CONF_PORT],
```

Reply:
Yes, please change it.